packages/app/components/Header/HamburgerButton.tsx

```diff
@@ -0,0 +1,36 @@
+import HamburgerIcon from "../../public/assets/images/hamburger.svg";
+import CrossIcon from "../../public/assets/images/cross.svg";
+import Image from "next/image";
+
+interface HamburgerButtonProps {
+  open: boolean;
+  toggle: () => void;
+}
+
+export default function HamburgerButton({
```

Comment:
Let's rename this component to `MenuButton`, as it not only Hamburguer button. It's the Hamburguer and close icon together.